cypress/integration/13-contributionFlow.order.test.js

```diff
@@ -52,4 +52,39 @@ describe('Contribution Flow: Order', () => {
       cy.contains(`${user.firstName} ${user.lastName} is now a member of APEX's 'Sponsors' tier!`);
     });
   });
+
+  it('Can order with an existing orgnanization', () => {
+    cy.clock(Date.parse('2042/05/25'));
+
+    const visitParams = { onBeforeLoad: mockRecaptcha };
+    // Login and redirect to the order page
+    cy.login({ redirect: '/apex/contribute/tier/470-sponsors', visitParams }).then(() => {
+      // Select 'Test Collective' organization profile
+      cy.get('[type="radio"][value="10880"]').check();
```

Comment:
Relying on the id (`[value="10880"]`) is not generic (it will break if we create another collective in one of the previous tests) so we should select the entry based on the label - either by having a random organization name to ensure it's unique or by using `:first`/`:last` selectors.

Reply:
@Betree I created a new organization and used the organization id as `([value=collective.id`), I think this is more reliable?

Reply:
@flickz It's a great way to solve it indeed